### PR TITLE
added test cases for OwnerRepo function

### PR DESCRIPTION
**Pull request by roz3x**
Signed-off-by: roz3x <oroz3x@gmail.com>

### CODE REVIEW

File: utils/repository.go [hunk 1]
```diff
@@ -15,13 +15,21 @@ func OwnerRepo(repoName string) (string, string, *errors.Error) {
 		return "", "", errors.New("parsing repository name: invalid number of parts")
 	}
 
+	errInvalidCharacters := errors.New("repository name contains invalid characters")
```
Comment (erikh) [inline utils/repository.go]: put this near the top of the file as a `var` declaration and make it public (capitalized). this way other consumers can use it.

Comment (roz3x) [reply utils/repository.go]: should i also do  this for other errors in file ?

**Comment by erikh** [comment]
Not right now, we'll do that later.

>

**Comment by erikh** [comment]
sorry to ask, but can you replace the current commit message with something like "added test cases for OwnerRepo function"?

**Pull request merged**